Appraisals

```diff
@@ -10,6 +10,10 @@ appraise 'rails-8.1' do
   gem 'rails', '~> 8.1.0.rc1'
 end
 
+appraise 'rails-8.2' do
```

Comment:
for now you can just remove this piece and it should be ok. once there's an actual branch/gem release this can be added back.